### ISSUE TITLE
Background color for screenshot function

### ISSUE THREAD

**Issue by ch-schiffer** [FZJ-INM1-BDA/siibra-explorer#1193]
It would be nice if the user could choose the background color when taking screenshots in the viewer, including a transparent background. This would make it much easier to use images created using the screenshot function (e.g., in presentations). In particular, it's difficult to create nice images of templates with black background (e.g., Colin27).

**Comment by xgui3783** [comment]
@fsdavid aren't you working on a feature that allow user change the background of viewer? can you tag the PR here?

**Comment by xgui3783** [comment]
found it:

https://github.com/FZJ-INM1-BDA/siibra-explorer/pull/1153

also, duplicate of https://github.com/FZJ-INM1-BDA/siibra-explorer/issues/1131 ?

**Comment by fsdavid** [comment]
Yes, but it is only about changing the color of the transparent background of the perspective view. 
Changing the background color of the slice view would be like this:
![20221028_124_4756b5a2](https://user-images.githubusercontent.com/26521731/198561845-3143969f-3c7e-409a-b5a3-d67cefb892b2.png)


https://github.com/FZJ-INM1-BDA/siibra-explorer/issues/1131 - Same PR could fix both.

**Comment by xgui3783** [comment]
I see. 

We might be able to do this another way. Afterall, changing all pixel from one color to another can be relatively easily achieved.

**Comment by fsdavid** [comment]
Sure, I just don't know how it would affect the brain image itself (if there are any black pixels).

**Comment by xgui3783** [comment]
Can't be helped, I guess?

**Comment by fsdavid** [comment]
Yes, I think so as well. If there is no other way.
